tests/php/Security/MemberTest.yml

```diff
@@ -58,6 +58,8 @@
     Email: noexpiry@silverstripe.com
     Password: 1nitialPassword
   staffmember:
+    FirstName: Staff
+    Surname: User
```

Comment:
I've confirmed that this doesn't affect any other tests - but it _is_ necessary for the new one, or the value in the map will just be whitespace.